subprojects/core/src/main/java/org/gradle/api/internal/project/taskfactory/AnnotationProcessingTaskFactory.java

```diff
@@ -52,16 +51,6 @@ public ITaskFactory createChild(ProjectInternal project, InstantiationScheme ins
     private <S extends Task> S process(S task) {
         TaskClassInfo taskClassInfo = taskClassInfoStore.getTaskClassInfo(task.getClass());
 
-        if (taskClassInfo.isIncremental()) {
```

Comment:
This is now unused, and a fair bit of code can be removed with it.

Reply:
Well, not much, but a small bit can at least be removed.

Reply:
I removed the property `incremental`.